tests/run-tests.py

```diff
@@ -0,0 +1,130 @@
+import contextlib
+import logging
+import os
+import subprocess as sp
```

Comment:
I know this is a nitpick, but this really confuses me and make code harder to read for others. Sorry...

Reply:
done